Mathlib/CategoryTheory/Functor/KanExtension/Adjunction.lean

```diff
@@ -136,6 +167,47 @@ noncomputable def lanCompColimIso (L : C ⥤ D) [∀ (G : C ⥤ H), L.HasLeftKan
       congr 1
       exact congr_app (L.lanUnit.naturality f) i))
 
+variable [HasColimitsOfShape D H]
+
+variable (G : C ⥤ H) [L.HasPointwiseLeftKanExtension G]
+
+omit [∀ (F : C ⥤ H), L.HasLeftKanExtension F]
+
+@[local instance]
+private theorem colimitIsoColimitGrothendieck_inst₁ :
+    HasColimit (CostructuredArrow.grothendieckProj L ⋙ G) :=
+  hasColimit_of_hasColimit_fiberwiseColimit_of_hasColimit _
+
+variable [HasColimitsOfShape C H]
+
+/-- If `G : C ⥤ H` is a left Kan extension of a functor `L : C ⥤ D` and `H` has colimits of shape
```

Comment:
This docstring doesn't really make sense to me, are you sure' it's correct?

Reply:
Not sure what you mean, should i mention `CostructuredArrow.grothendieckProj` explicitly instead of just saying "a canonical functor ..."?

Reply:
Shouldn't the beginning of the sentence be more like "if `G : C => H` admits a pointwise left Kan extension along `L : C => H` and ..."?

Reply:
Ah, right, I always autocorrected that in my head when I read it